src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,49 @@
+.catalog {
+  &__container {
+
+    padding: #{$catalog-container-top-padding}
+      #{$catalog-container-hor-min-padding}
+      #{$catalog-container-bottom-padding}
+    ;
+
+    min-width: getWidth(
+      $card-width,
+      $catalog-card-hor-right-margin,
+      $catalog-container-hor-min-padding
+    );
+    max-width: getWidth(
+      $card-width,
+      $catalog-card-hor-right-margin,
+      $catalog-container-hor-max-padding
+    );
+
+    margin: 0 auto;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: flex-start;
+    flex-wrap: wrap;
+
+    @media (min-width: 1200px) {
+      & {
+        padding: #{$catalog-container-top-padding}
+          #{$catalog-container-hor-max-padding}
+          #{$catalog-container-bottom-padding}
+        ;
+      }
+    }
```

Comment:
i don't believe you need this entire `@media` - just provide your cards container with a fixed width of 1024px and you should be good
now at 1200px you have this:
<img width="623" alt="Monosnap Catalog 2022-11-10 16-54-18" src="https://user-images.githubusercontent.com/39308717/201124663-4e8a57f4-91df-4493-8924-0060079244e1.png">

Reply:
You're right, everything continuous working well if I remove the media.